roles/glusterfs/peer_probe/tests/test.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: servers
```

Comment:
In the inventory you have [master] and here you are mentioning the hosts as servers. 

In peer probe, we do not run the probe from all the servers. Only from master.

Reply:
@sac made the suggested changes. can you please retry.
Thanks for the review :)